static/webpack/getSecuredJamboInjectedData.js

```diff
@@ -0,0 +1,32 @@
+const _ = require('lodash');
+
+/**
+ * Returns JAMBO_INJECTED_DATA with instances of the global config's apiKey removed
+ * 
+ * @param {Object} data JAMBO_INJECTED_DATA
+ * @returns {Object}
+ */
+function getSecuredJamboInjectedData (data) {
```

Comment:
Small nit: The phrase "secured" implies certain things to me. For instance, it suggests that maybe that data is encrypted or something like that. What would you think of using another adjective? We're kind of "cleaning" the data here or "scrubbing" it.